.github/workflows/test.yaml

```diff
@@ -1,3 +1,10 @@
+name: 'CI'
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
```

Comment:
Is this correct `yml` syntax here? Leaving the colon without anything inside (beneath) it?

Reply:
it's what the github docs say to do. I believe it's equivalent to saying `pull_request: ~` and should deserialize to

```
{"pull_request": None}
```

Reply:
Perfect! Thanks